misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmd.java

```diff
@@ -112,6 +112,7 @@ public void execute() {
         final File openApiFile = new File(argList.get(0));
         final String openApiFilePath = openApiFile.getPath();
         Path resourcePath = Paths.get(resourcesDirectory + "/" + openApiFile.getName());
+        Path relativeResourcePath = Paths.get( "resources", openApiFile.getName());
```

Comment:
remove unnecessary space in front of "resources" literal.

Reply:
Fixed efaff44cdb0fb731ba76f610f9ead17266326de2